docker/worker/worker_test.py

```diff
@@ -609,6 +609,12 @@ def setUp(self):
 
     osv.ecosystems.work_dir = 'testdata/tmp/'
 
+    # add fake ecosystems used in tests to supported ecosystems
+    osv.ecosystems._ecosystems.update({
+        'ecosystem': osv.ecosystems.OrderingUnsupportedEcosystem(),
+        'golang': osv.ecosystems.OrderingUnsupportedEcosystem(),
```

Comment:
Can you update the tests instead of adding this `golang` one here? The correct ecosystem is "Go".